packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts

```diff
@@ -43,7 +42,7 @@ export const Container = styled.div`
   top: var(--size-topbar-height);
   left: 0;
   right: 0;
-  height: calc(100vh - var(--size-topbar-height));
+  height: calc(100% - var(--size-topbar-height));
```

Comment:
This should work since the parent will probably remain a flex element but alternatively this is what [`dvh`](https://caniuse.com/mdn-css_types_length_viewport_percentage_units_dynamic) was meant to solve